app.js

```diff
@@ -0,0 +1,99 @@
+
+const path = require('path');
+const uuid = require('uuid');
+
+const express = require('express');
+const session = require('express-session');
+const bodyParser = require('body-parser');
+
+const RedisStore = require('connect-redis')(session);
+const passport = require('passport');
+const OnshapeStrategy = require('passport-onshape');
+const fetch = require('node-fetch');
+
+const redisClient = require('./redis-client');
+
+const { onshapeApiUrl } = require('./utils');
+
+const app = express();
+
+/*app.set('views', path.join(__dirname, 'views'));
+app.engine('html', require('hbs').__express);
+app.set('view engine', 'html');*/
```

Comment:
Don't checkin commented out code please.

Reply:
Removed in new push.